tests/game_test.py

```diff
@@ -74,3 +78,22 @@ def test_get_unconfirmed_move(fx_session, fx_user, fx_novice_status):
     move.id = ('00000000000000000000000000000000'
                '00000000000000000000000000000000')
     assert not get_unconfirmed_move(fx_user.address)
+
+
+def test_prevent_hack_and_slash_when_dead(
+        fx_test_client: Client, fx_session: Session, fx_user: User,
```

Comment:
Actually `Client` is not `werkzeug.test.Client`, but `flask.testing.FlaskClient`.

Reply:
👌